assets/src/dashboard/components/viewStyleBar/index.js

```diff
@@ -61,33 +58,23 @@ const ToggleButton = styled.button`
   }
 `;
 
-const ListIcon = styled(ListSVG).attrs(ICON_METRICS.VIEW_STYLE)`
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray500};
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-`;
-
-const GridIcon = styled(GridSVG).attrs(ICON_METRICS.VIEW_STYLE)`
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray500};
-  display: flex;
-  justify-content: flex-start;
-  align-items: center;
-`;
-
 export default function ViewStyleBar({ onPress, layoutStyle }) {
   return (
     <Container>
-      <Tooltip content={VIEW_STYLE_LABELS[layoutStyle]} position="right">
+      <Tooltip
+        title={VIEW_STYLE_LABELS[layoutStyle]}
+        placement={TOOLTIP_PLACEMENT.BOTTOM_END}
+        hasTail
+      >
         <ToggleButton
```

Comment:
should we update the toggle button to use the new `Button`?

Reply:
It's a bit different of a toggle, it's actually just replacing the view.